pyflowater/__init__.py

```diff
@@ -300,3 +301,11 @@ def consumption(self, device_id, startDate=None, endDate=None, interval=INTERVAL
         
         url = f"{FLO_V2_API_BASE}/water/consumption"
         return self.query(url, method=METHOD_GET, extra_params=params)
+
+    def get_real_time_listener(self, mac_address, callback):
+        """Begin listening for the specified device (note MAC address, not ID), sending results to the specified callback.
+
+        Returned FloListener must be manually started."""
```

Comment:
Callback is a function of one argument that will be called with a dictionary of the form [...]

Returns a FloListener. You must call start() on the returned FloListener in order to start getting callbacks.